Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/StatusBarHeight.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.ManualReview)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.None, 20090125, "Get height of the status bar", PlatformAffected.iOS | PlatformAffected.Android)]
+	public class StatusBarHeightTest : TestContentPage // or TestMasterDetailPage, etc ...
+	{
+		protected override void Init()
+		{
+			// Initialize ui here instead of ctor
+			Content = new Label
+			{
+				AutomationId = "StatusBarHeight",
+				Text = $"The status bar is {Device.Info.StatusBarHeight} units tall. "
+					+ Environment.NewLine + "iOS should be 44."
+					+ Environment.NewLine + "Android should be 24.,
+				HorizontalOptions = LayoutOptions.Center,
+				VerticalOptions = LayoutOptions.Center
+			};
+		}
+
+#if UITEST
+		[Test]
+		public void StatusBarHeightTest() 
+		{
```

Comment:
Fix the test it will always fail

Reply:
ok, not sure how to test it so I remove it. lmk if you have thoughts on how to test.